go.mod

```
@@ -164,7 +165,7 @@ replace (
 	// See https://github.com/tecbot/gorocksdb/pull/216
 	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.1.1
 
-	github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.10.0-alpha1-cronos-4
+	github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.10.0-alpha1-cronos-5
```

```suggestion
	github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.10.0-alpha1-cronos-5
```
TODO, update after pass integration test.

---

https://github.com/yihuang/ethermint/compare/v0.10.0-alpha1-cronos-4...v0.10.0-alpha1-cronos-5